src/main/scala/com/databricks/spark/csv/util/TypeCast.scala

```diff
@@ -35,7 +38,8 @@ object TypeCast {
    * @param datum string value
    * @param castType SparkSQL type
    */
-  private[csv] def castTo(datum: String, castType: DataType, nullable: Boolean = true): Any = {
+  private[csv] def castTo(datum: String, castType: DataType, nullable: Boolean = true,
+                          locale: Locale = Locale.getDefault): Any = {
```

Comment:
You are right,  I add this parameter just for testing, because if I use default Locale  in mi local environment test assert(TypeCast.castTo("1,00", FloatType) == 1.0) pass but travis fails because its locale its different.  What do you propose to solve this?